internal/cmd/plugin/logical/publication/create/cmd.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright The CloudNativePG Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package create
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cloudnative-pg/cloudnative-pg/internal/cmd/plugin/logical"
+)
+
+// NewCmd initializes the publication create command
+func NewCmd() *cobra.Command {
+	var dbName string
+	var allTables bool
+	var schemaNames []string
+	var tableExprs []string
+	var publicationName string
+	var externalClusterName string
+	var dryRun bool
+
+	publicationCreateCmd := &cobra.Command{
+		Use:   "create cluster_name",
+		Args:  cobra.ExactArgs(1),
+		Short: "create a logical replication publication",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			dbName := strings.TrimSpace(dbName)
```

Comment:
`dbName` should not be mandatory. We should allow users to pass the `--dbname` option, while defaulting to `app`.